src/main/scala/com/fulcrumgenomics/bam/ErrorRateByReadPosition.scala

```diff
@@ -0,0 +1,170 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+package com.fulcrumgenomics.bam
+
+import com.fulcrumgenomics.cmdline.{ClpGroups, FgBioTool}
+import com.fulcrumgenomics.util.{Metric, NumericCounter, ProgressLogger}
+import dagr.commons.CommonsDef._
+import dagr.commons.io.{Io, PathUtil}
+import dagr.commons.util.LazyLogging
+import dagr.sopt._
+import htsjdk.samtools.util.SequenceUtil
+import htsjdk.samtools.{SAMRecord, SamReaderFactory}
+
+/**
+  * Calculates the error rate by position within each read.
+  *
+  * Gives the error rate for read one, read two, and across both reads, for the given position relative to the start of
+  * the read.  For paired end reads, the read position is relative to the first base sequenced for the given end of the
+  * read.  Soft-clipped bases, and unmapped, secondary, supplementary, duplicate, and vendor failing reads are ignored.
+  *
+  * @author Nils Homer
+  */
+@clp(description =
+  """
+    |Calculates the error rate by read position.
+  """,
+  group = ClpGroups.SamOrBam)
+class ErrorRateByReadPosition
```

Comment:
Agreed!  But I don't need it for my current purposes, so lets add this feature in when it is needed.